### PR TITLE
Feat - invite user to team flow improvements

### LINKED ISSUE #6577: [CAL-833] Improve Accepting Team Invite Flow
As no team member received the confirmation email, I asked them to accept the invitation through the app, as they appeared as their status was “Pending”.

Some of our team members suggested that the option to accept the invitation could be in a more visible or direct place. Probably right after logging and entering to the app?

Created via Threads. See full discussion: [https://threads.com/34443144252](https://threads.com/34443144252)

<sub>From [SyncLinear.com](https://synclinear.com) | [CAL-833](https://linear.app/calcom/issue/CAL-833/improve-accepting-team-invite-flow)</sub>

### PR DESCRIPTION
**Pull request by sean-brydon**
## What does this PR do?

Fixes: #6577 
Existing User
CleanShot 2023-01-26 at 16 29 32@2x

New User (Accidentally highlighted the text oops)
CleanShot 2023-01-26 at 16 29 09@2x

https://www.loom.com/share/fc8011ea5939431a90e3f1d97dc19099

### CODE REVIEW

File: packages/emails/src/components/CallToAction.tsx [hunk 1]
```diff
@@ -14,7 +14,7 @@ export const CallToAction = (props: { label: string; href: string; secondary?: b
       margin: 0,
       textDecoration: "none",
       textTransform: "none",
-      padding: "10px 16px",
+      padding: "16px 24px",
```
Comment (sean-brydon) [inline packages/emails/src/components/CallToAction.tsx:17]: Bigger CTA button as per designs

File: packages/emails/src/components/V2BaseEmailHtml.tsx [hunk 2]
```diff
@@ -26,8 +26,8 @@ export const V2BaseEmailHtml = (props: {
   return (
     <Html>
       <EmailHead title={props.subject} />
-      <body style={{ wordSpacing: "normal", backgroundColor: "#F5F5F5" }}>
-        <div style={{ backgroundColor: "#F5F5F5" }}>
+      <body style={{ wordSpacing: "normal", backgroundColor: "#F3F4F6" }}>
+        <div style={{ backgroundColor: "#F3F4F6" }}>
```
Comment (sean-brydon) [inline packages/emails/src/components/V2BaseEmailHtml.tsx:30]: was using old gray 100

File: packages/features/ee/teams/components/MemberInvitationModal.tsx [hunk 3]
```diff
@@ -153,7 +153,7 @@ export default function MemberInvitationModal(props: MemberInvitationModalProps)
               color="primary"
               className="ltr:ml-2 ltr:mr-2 rtl:ml-2"
               data-testid="invite-new-member-button">
-              {t("invite_new_member")}
+              {t("invite")}
```
Comment (sean-brydon) [inline packages/features/ee/teams/components/MemberInvitationModal.tsx:156]: This invite_new_member text was far too long

File: packages/trpc/server/routers/viewer/teams.tsx [hunk 4]
```diff
@@ -291,7 +289,7 @@ export const viewerTeamsRouter = router({
           });
 
         // valid email given, create User and add to team
-        const user = await ctx.prisma.user.create({
+        await ctx.prisma.user.create({
```
Comment (sean-brydon) [inline packages/trpc/server/routers/viewer/teams.tsx:292]: We were never doing anything with user so just running function

File: packages/emails/src/templates/TeamInviteEmail.tsx [hunk 5]
```diff
@@ -22,39 +24,100 @@ export const TeamInviteEmail = (
...
-      <CallToAction label={props.language("accept_invitation")} href={props.joinLink} />
+
+      {!props.isCalcomMember && (
+        <div style={{ display: "flex", flexDirection: "column", gap: "32px" }}>
+          <EmailStep
+            translationString={props.language("email_no_user_step_one")}
+            iconsrc={
+              IS_PRODUCTION
+                ? WEBAPP_URL + "/emails/choose-username@2x.png"
+                : "http://localhost:3000/emails/choose-username@2x.png"
```
Comment (alannnc) [inline packages/emails/src/templates/TeamInviteEmail.tsx:79]: Will this work for self hosted?

Comment (sean-brydon) [reply packages/emails/src/templates/TeamInviteEmail.tsx:79]: It should? If webapp_url is set... This is how we handled images in the past in email so if not they wont work either :D

**Review by alannnc — COMMENTED** [review]
Code looks good, just not so sure about hardcoded http://localhost:3000/ url.

**Review by alannnc — APPROVED** [review]

**Review by PeerRich — APPROVED** [review]

**Comment by shirazdole** [comment]
@sean-brydon - we should try to avoid contractions in formal communications from Cal.com. I.e. "we'll" should be "we will" and "you'll" should be "you will" 🙏🏽 "you've" -> "you have" 

There should be a period at the end of "Contact our support team." and an exclamation at the end of "Happy scheduling from the Cal.com team!" 

Overall, great work 🚀

**Comment by shirazdole** [comment]
cc @Jaibles 😅

**Comment by Jaibles** [comment]
Sounds like we need a "Tone of voice" guidelines doc?

**Pull request merged**